web/gui/dashboard_info.js

```diff
@@ -3997,6 +4001,22 @@ netdataDashboard.context = {
         info: 'The function <code>swap_writepage</code> is called when the kernel writes a page to swap memory.'
     },
 
+    'apps.shmget_call': {
+        info: 'Number of times the syscall <code>shmget</code> is called. Netdata also gives a summary for these charts in <a href="#menu_system_submenu_shared_memory_calls">System overview</a>.'
```

Comment:
On my dashboard the link for `system overview` did not work, please, update it. Considering what I saw during my tests, we need to use the anchor `menu_system_submenu_ipc_shared_memory`.

Reply:
Oh, so sorry about that :man_facepalming: